modules/i3s/src/i3s-loader.js

```diff
@@ -46,6 +50,7 @@ const I3SLoader = {
       loadContent: true,
       isTileset: 'auto',
       isTileHeader: 'auto',
+      isNodePage: false,
```

Comment:
I am not a big fan of these `isTileset`, `isNodePage` flags... Is there anyway we can detect this?

Maybe create a separate loader?

Reply:
Done